src/Templating/Helper/BlockHelper.php

```diff
@@ -341,38 +341,9 @@ protected function computeAssets(BlockContextInterface $blockContext, array &$st
         ];
     }
 
-    private function startTracing(BlockInterface $block): array
-    {
-        if (null !== $this->stopwatch) {
-            $this->traces[$block->getId()] = $this->stopwatch->start(
-                sprintf('%s (id: %s, type: %s)', $block->getName(), $block->getId(), $block->getType())
-            );
-        }
-
-        return [
-            'name' => $block->getName(),
-            'type' => $block->getType(),
-            'duration' => false,
-            'memory_start' => memory_get_usage(true),
-            'memory_end' => false,
-            'memory_peak' => false,
-            'cache' => [
-                'keys' => [],
-                'contextual_keys' => [],
-                'handler' => false,
-                'from_cache' => false,
-                'ttl' => 0,
-                'created_at' => false,
-                'lifetime' => 0,
-                'age' => 0,
-            ],
-            'assets' => [
-                'js' => [],
-                'css' => [],
-            ],
-        ];
-    }
-
+    /**
+     * @internal since sonata-project/block-bundle 3.16
```

Comment:
If these `@internal` annotations (which were already resolved using private visibility in 4.x) are intended to be removed in another PR, I think this one is ready.

Reply:
Yeah that's for the cleanup PR I guess #602